README.md

```diff
@@ -43,7 +44,7 @@ If you are new to Graviton and want to understand how to identify target workloa
 |Core	|[Neoverse-N1](https://developer.arm.com/documentation/100616/0301)	|[Neoverse-V1](https://developer.arm.com/documentation/101427/latest/)	|
 |Frequency	|2500MHz	|2600MHz	|
 |Turbo supported	|No	|No	|
-|Instruction latencies	|[Instruction Latencies](https://developer.arm.com/documentation/PJDOC-466751330-9707/r4p1/?lang=en)	|[Instruction Latencies](https://developer.arm.com/documentation/pjdoc466751330-9685/latest/)	|
+|Software Optimization Guide (Instruction Throughput and Latency)|[SWOG](https://developer.arm.com/documentation/PJDOC-466751330-9707/r4p1/?lang=en)|[SWOG](https://developer.arm.com/documentation/pjdoc466751330-9685/latest/) |
```

Comment:
Can both links be to the latest version?

Reply:
V1 was up to date, but I updated N1.